vite.config.ts

```diff
@@ -19,38 +20,39 @@ const resizeColumn: BrowserCommand<[resizeBy: number]> = async (context, resizeB
   await page.mouse.up();
 };
 
-export default defineConfig({
-  base: isCI ? '/react-data-grid/' : '/',
+export default defineConfig(({ command }) => ({
+  base: '/react-data-grid/',
   cacheDir: '.cache/vite',
+  clearScreen: false,
   build: {
-    emptyOutDir: true,
-    sourcemap: true
+    target: browserslist().map((version) => version.replace(' ', '')),
+    modulePreload: { polyfill: false },
```

Comment:
This will optimize the build output

Reply:
Which polyfills were getting included in the build?

Reply:
Nothing in the RDG output, but I remember seeing differences in other projects.
I think it'd matter if we had lazy-loaded modules, so it'd use native methods to pre-load split chunks.